model/project_vars_test.go

```diff
@@ -1164,13 +1164,15 @@ func TestFullSyncToParameterStore(t *testing.T) {
 			}
 			require.NoError(t, db.Insert(ProjectVarsCollection, projVars))
 
-			require.NoError(t, FullSyncToParameterStore(ctx, &projVars, &projRef, false))
+			pm, err := FullSyncToParameterStore(ctx, &projVars, &projRef, false)
+			require.NoError(t, err)
 
 			checkAndSetProjectVarsSynced(t, &projRef, false)
 
 			dbProjVars, err := FindOneProjectVars(projVars.Id)
 			require.NoError(t, err)
 			require.NotZero(t, dbProjVars)
+			dbProjVars.Parameters = *pm
```

Comment:
should `require.NotNil(t, pm)` here (same for L1206)

Reply:
This is already checked by `require.NoError` above because L1167 never returns nil, nil (even if there's no parameters to sync).

In any case, this test is going to go away after the migration because the full sync is just a temporary migration feature. I really just wrote the tests as a spot check for myself to verify that the logic I wrote made sense/caught basic cases.